src/audiovideocontroller/AudioVideoControllerState.ts

```diff
@@ -201,6 +201,8 @@ export default class AudioVideoControllerState {
   // and subscribe-ack)
   isSessionConnected: boolean = false;
 
+  retryCount: number = 0;
```

Comment:
Since we are moving to meeting session context now, do we have to reset this once reconnected? Earlier it was tied to the DefaultACController, not the shared context hence checking.